webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -286,6 +286,55 @@ test.describe('Experiment List', () => {
     );
   });
 
+  test('Multi-sort menu', async () => {
+    const getOrder = async (firstKey: keyof ExperimentBase, secondKey: keyof ExperimentBase) => {
```

Comment:
how about well call this `checkOrder` instead and always expect the first key to be first?

Reply:
i think a change like this would help a lot with clarity at a glance